maintainers-guide.md

```diff
@@ -0,0 +1,210 @@
+---
+layout: contribute
+title: Guide for Bazel Maintainers
+---
+
+# Guide for Bazel Maintainers
+
+This is a guide for the maintainers of the Bazel open source project.
+
+If you are looking to contribute to Bazel, please read [Contributing to
+Bazel](/contributing.html) instead.
+
+The objectives of this document are to:
+
+1. Serve as the maintainers' source of truth for the project’s contribution
+   process.
+1. Set expectations between the community contributors and the project
+   maintainers.
+   
+<!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
```

Comment:
As much as I love Emacs, I don't think everyone is using it.
Shall we use Kramdown like on the docs site?

Reply:
Opened https://github.com/bazelbuild/bazel-website/pull/150 to split Kramdown integration from this PR.

Reply:
Switched to kramdown's toc generation macro.